tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -0,0 +1,346 @@
+    user opens accordion section    %{TEST_THEME_NAME}
+    user waits until accordion section contains text    %{TEST_THEME_NAME}    %{TEST_TOPIC_NAME}
+
+    user opens details dropdown    %{TEST_TOPIC_NAME}
+
+    user waits until details dropdown contains publication    %{TEST_TOPIC_NAME}    ${PUBLICATION_NAME_SUPERSEDE}
+    user checks publication bullet contains link    ${PUBLICATION_NAME_SUPERSEDE}    View statistics and data
+
+    user waits until details dropdown contains publication    %{TEST_TOPIC_NAME}    ${PUBLICATION_NAME_ARCHIVE}
+    user checks publication bullet contains link    ${PUBLICATION_NAME_ARCHIVE}    View statistics and data
+
+Check public archive-publication release page displays correctly after being unarchived
+    user clicks element    testid:View stats link for ${PUBLICATION_NAME_ARCHIVE}
+
+    user waits until h1 is visible    ${PUBLICATION_NAME_ARCHIVE}    %{WAIT_MEDIUM}
+    user waits until page contains    This is the latest data
+
+Check public data tables page is correct after archive-publication has been unarchived
+    user navigates to data tables page on public frontend
+
+    user opens details dropdown    %{TEST_THEME_NAME}
+    user opens details dropdown    %{TEST_TOPIC_NAME}
+
+    user checks page contains    ${PUBLICATION_NAME_ARCHIVE}
+
+    user clicks radio    ${PUBLICATION_NAME_ARCHIVE}
+    user clicks element    id:publicationForm-submit
+    user waits until table tool wizard step is available    2    Choose a subject
+    user checks previous table tool step contains    1    Publication    ${PUBLICATION_NAME_ARCHIVE}
+
+    user checks page contains    ${SUBJECT_NAME_ARCHIVE}
+
+Check data catalogue page is correct after archive-publication has been unarchived
+    user navigates to data catalogue page on public frontend
+
+    user opens details dropdown    %{TEST_THEME_NAME}
+    user opens details dropdown    %{TEST_TOPIC_NAME}
+
+    user checks page contains    ${PUBLICATION_NAME_ARCHIVE}
+    user checks page contains    ${PUBLICATION_NAME_SUPERSEDE}
+
+    user clicks radio    ${PUBLICATION_NAME_ARCHIVE}
+    user clicks button    Next step
+    user waits until page contains    Choose a release
+    user waits until page contains    ${RELEASE_NAME_ARCHIVE}
+
+    user checks page contains radio    ${RELEASE_NAME_ARCHIVE}
+    user checks page contains    This is the latest data
+
+    user clicks button    Previous step
+    user waits until page contains    ${PUBLICATION_NAME_SUPERSEDE}
+
+    user clicks radio    ${PUBLICATION_NAME_SUPERSEDE}
+    user clicks button    Next step
+    user waits until page contains    Choose a release
+    user waits until page contains    ${RELEASE_NAME_SUPERSEDE}
+
+    user checks page contains radio    ${RELEASE_NAME_SUPERSEDE}
+    user checks page contains    This is the latest data
+
+Check archive-publication permalink no longer has out-of-date warning after archive-publication has been unarchived
+    go to    ${PERMALINK_URL}
```

Comment:
I believe you'll need to use the user goes to public frontend KW in order to avoid issues with how basic auth works on the public FE